specs/HubProtocol.md

```diff
@@ -16,7 +16,7 @@ The SignalR Protocol requires the following attributes from the underlying trans
 
 ## Overview
 
-There are two encodings of the SignalR protocol: [JSON](http://www.json.org/) and [Protocol Buffers](https://developers.google.com/protocol-buffers/). Only one format can be used for the duration of a connection, and the format must be negotiated in advance (i.e. using a QueryString value, Header, or other indicator). However, each format shares a similar overall structure.
```

Comment:
I am not sure if we want to remove protobuf from the spec or just move it somewhere - it still can be useful if we want to add a sample ProtoBufHubProtocol.